mods/ra/rules/fakes.yaml

```diff
@@ -393,3 +393,39 @@ FACF:
 		Type: Rectangle
 			TopLeft: -1536, -1536
 			BottomRight: 1536, 1536
+
+TENF:
+	Inherits: ^FakeBuilding
+	Inherits@infiltrate: ^InfiltratableFake
+	Inherits@shape: ^2x2Shape
+	Selectable:
+		Bounds: 48,48
+	Buildable:
+		BuildPaletteOrder: 871
```

Comment:
Please adjust the `BuildPaletteOrder`s here and in the other fake structures to follow the standard convention - see previous discussion in #16613 for reasoning.

Reply:
Updated. As it seems there was already a place for it